projects/ngx-drag-to-select/src/lib/select-container.component.ts

```diff
@@ -468,6 +475,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
     if (!this._hasItem(item, selectedItems)) {
       success = true;
       selectedItems.push(item.value);
+      this.itemSelected.emit(item.value);
```

Comment:
Out of curiosity, would it make a difference to emit this after `this._selectedItems$.next(selectedItems)`?

Reply:
Because then it's "really" select**ed** and deselect**ed**.